src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -57,8 +58,12 @@ class Serializer implements SerializerInterface, NormalizerInterface, Denormaliz
      * @var array
      */
     protected $denormalizerCache = array();
+    /**
+     * @var PostNormalizerInterface[]
+     */
+    protected $postNormalizers = array();
```

Comment:
please make it private (all properties should start their life as private until we make an explicit decision to turn them into inheritance-based extension points)